CMakeLists.txt

```diff
@@ -103,7 +103,15 @@ IF(NOT CYCLUS_DOC_ONLY)
     # Include the boost header files, system, and filesystem libraries
     SET(Boost_USE_STATIC_LIBS       OFF)
     SET(Boost_USE_STATIC_RUNTIME    OFF)
-    FIND_PACKAGE(Boost COMPONENTS filesystem system REQUIRED)
```

Comment:
the explicit `BOOST` detection was actually there... I just modify it to match the Cyclus one